tsearch_extras.c

```diff
@@ -74,9 +77,13 @@ ts_match_locs_setup(Oid cfgId, TsMatchesData *mdata, text* in, TSQuery query)
 
 	hlparsetext(cfgId, &prs, query, VARDATA(in), VARSIZE(in) - VARHDRSZ);
 
+	headline_options = lappend(headline_options,
+											  makeDefElem(pstrdup("HighlightAll"),
+											  (Node *) makeString(pstrdup("1"))));
```

Comment:
This seems way over-indented, doesn't it?

Reply:
That's how the rest of the file is indented using tabs (https://github.com/sampritipanda/tsearch_extras/blob/a4277cb22a918dfc6bc2fd4009f75c2152f2ad83/tsearch_extras.c#L171)

Should I convert all of it to spaces?

Reply:
Oh, I see.  Hmm, we probably shouldn't change it, since I bet the code has the same style rules as Postgres upstream.